lib/ruby-measurement/definitions/us_customary/volume.rb

```diff
@@ -8,6 +8,8 @@
   unit.convert_to(:'fl oz') { |value| value * 128.0 }
   unit.convert_to(:tbsp) { |value| value * 256.0 }
   unit.convert_to(:tsp) { |value| value * 768.0 }
+  unit.convert_to(:ml) { |value| value * 3785.411784 }
+  unit.convert_to(:l) { |value| value * 3.785411784 }
```

Comment:
My understanding of your approach was to allow people to selectively `require` source units (i.e. what they were converting *from*) but, once that was required, to allow conversion *to any* destination units of the same type (e.g. volume :arrow_right: volume). 

I think it’s quite common to want to convert units across the US/metric boundary and I’d hope this library would allow that functionality, even if that means offering a less granular `require` options, the value of which is less obvious, at least to me.